packages/amplify-category-analytics/tests/commands/analytics.test.js

```diff
@@ -0,0 +1,24 @@
+const analyticsHelp = require('../../commands/analytics');
+
+describe('analytics help: ', () => {
+  const mockRemoveResource = jest.fn(() => Promise.resolve());
+  const mockRemoveResourceWithError = jest.fn(() => Promise.reject('Error'));
+  const mockContext = {
+    amplify: {
+      showHelp: jest.fn(),
+    },
+    parameters: {
+      first: 'analytics',
+    },
+    print: {
+      info: jest.fn().mockImplementation(info => console.log(info)),
```

Comment:
Do we need mock implentation? Do we really care what gets printed on the console?